internal/manifests/collector/container_test.go

```diff
@@ -70,12 +95,14 @@ func TestContainerPorts(t *testing.T) {
 	var goodConfig = `receivers:
   examplereceiver:
     endpoint: "0.0.0.0:12345"
+exporters:
+  logging:
```

Comment:
this should be called the debug exporter, docs [here](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/debugexporter)